docs/man5/Makefile.am

```diff
@@ -3,7 +3,8 @@ MAN5_FILES  = \
 
 A2X_ARGS = \
 	-d manpage \
-	-f manpage
+	-f manpage \
+	-L
```

Comment:
On OSX, xmllint fails to load the docbook dtd (and --nonet is specified so it does not try to fetch it). This is apparently because the OSX package for xmllint does not include these standard DTDs. If you remove "--nonet" from the xmllint options, the command succeeds because it will go out and download the DTD. However this is not ideal as it requires net access for the build.

Best option seems to be disabling xmllint based on a conditional -- this can be OS-based or by asking xmllint to validate a dummy xml file against the docbook DTD.